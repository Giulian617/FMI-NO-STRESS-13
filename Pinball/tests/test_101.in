100 0
71 26 8
26 77 6
12 26 3
86 26 8
26 100 4
26 15 4
26 68 8
66 26 2
26 63 4
26 37 3
84 26 1
40 26 7
42 26 10
26 28 7
50 26 3
30 26 10
26 29 8
57 26 2
26 14 3
26 51 1
39 26 9
62 26 10
53 26 9
26 9 4
88 26 5
7 26 7
73 26 9
26 11 2
96 26 2
67 26 7
74 26 7
26 56 10
52 26 3
26 41 8
93 26 5
26 75 8
64 26 10
26 44 8
36 26 10
26 31 8
26 34 2
24 26 4
25 26 10
78 26 7
3 26 5
26 17 6
59 26 3
26 83 10
26 65 5
26 76 10
82 26 3
97 26 3
32 26 1
99 26 8
26 2 4
33 26 7
26 20 4
26 13 5
98 26 8
26 90 1
26 16 10
26 87 1
26 70 6
94 26 2
27 26 10
21 26 1
43 26 5
81 26 6
26 6 10
4 26 6
26 46 2
89 26 1
26 91 7
95 26 8
26 45 2
26 54 9
26 19 3
8 26 4
26 80 3
1 26 6
58 26 1
18 26 1
49 26 6
26 38 6
48 26 5
26 55 3
26 5 4
23 26 5
26 10 9
69 26 7
22 26 6
92 26 10
85 26 1
35 26 10
60 26 8
26 47 4
26 61 8
72 26 1
79 26 1
