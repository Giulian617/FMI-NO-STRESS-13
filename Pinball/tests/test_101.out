0
243992608
304990760
365988912
243992608
609981520
426987064
243992608
243992608
548983368
121996304
182994456
304990760
182994456
243992608
609981520
365988912
60998152
182994456
243992608
60998152
365988912
304990760
243992608
609981520
609981520
426987064
487985216
609981520
487985216
60998152
426987064
121996304
609981520
609981520
182994456
365988912
548983368
426987064
487985216
609981520
304990760
487985216
121996304
121996304
243992608
304990760
365988912
182994456
60998152
182994456
548983368
548983368
182994456
609981520
121996304
60998152
182994456
487985216
487985216
609981520
243992608
609981520
304990760
121996304
426987064
487985216
426987064
365988912
487985216
60998152
548983368
426987064
487985216
609981520
365988912
426987064
60998152
182994456
365988912
182994456
609981520
60998152
60998152
487985216
60998152
304990760
60998152
60998152
426987064
609981520
304990760
121996304
487985216
121996304
182994456
487985216
487985216
243992608
